### DISCUSSION TITLE
Dark color scheme support

### DISCUSSION THREAD

**Discussion by lucamegh** [pointfreeco/pointfreeco#912] [Ideas]
According to a [Medium article](https://medium.com/dev-channel/let-there-be-darkness-maybe-9facd9c3023d), around 80% of the developer community, myself included, use dark mode on their devices. Adding dark mode support to _pointfree.co_ would greatly enhance the user experience, making the website more enjoyable at night and better aligned with users' workspaces.

**Comment by acosmicflamingo** [comment]
I'm surprised if I search the word "accessibility" in that article, nothing appears. I'd think that the article would make the case for dark mode support for accessibility reasons, and not so much because it's an aesthetic preference for people.

**Comment by mbrandonw** [comment]
Hi @lucamegh, we have started implementing dark mode on Point-Free. There are more pages to go, but we are making progress.

**Reply by lucamegh** [reply]
Perfect timing! I love the new design. I’m excited to see dark mode on the episode page. Thanks!

**Reply by stephencelis** [reply]
This is now live for our latest episode: https://www.pointfree.co/episodes/ep286-modern-uikit-tree-based-navigation

We'll be back-filling past episodes over time.
